client/notices/global-notices.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import debugModule from 'debug';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import Notice from 'components/notice';
+const debug = debugModule( 'calypso:global-notices' );
+
+var NoticesList =  React.createClass( {
+
+	displayName: 'GlobalNotices',
```

Comment:
`displayName` is not necessary when assigning `React.createClass` to a variable, as it can be inferred from the variable name.